bin/node-template/node/src/service.rs

```diff
@@ -452,7 +453,7 @@ pub fn new_light(mut config: Configuration) -> Result<TaskManager, ServiceError>
 		);
 	}
 
-	sc_service::spawn_tasks(sc_service::SpawnTasksParams {
+	let _: RpcHandlers = sc_service::spawn_tasks(sc_service::SpawnTasksParams {
```

Comment:
If I remove them it doesn't compile, because the use of the default generic cannot be implied. You have to specify the type without specifying the CM type so that the default type is used. This is the least intrusive design I've found.